src/Explorer/Notebook/NotebookRenderer/OriginalPrompt.tsx

```diff
@@ -0,0 +1,66 @@
+// This is the original read-only [] prompt
```

Comment:
Can we use Prompt directly from Nteract? (instead of copying it)

Reply:
Good catch! I was having trouble importing because I was using the wrong src location (see your comment below). It now works with the correct imported package.